src/presenters/sessionUtilisateurPresenter.ts

```diff
@@ -27,3 +27,9 @@ export type SessionUtilisateurViewModel = Readonly<{
   }>
   uid: string
 }>
+
+const libelleByRole: Readonly<Partial<Record<TypologieRole, string>>> = {
+  Instructeur: 'Banque des territoires',
+  'Pilote politique publique': 'France Numérique Ensemble',
+  'Support animation': 'Mednum',
+}
```

Comment:
pourquoi ne pas mettre ça dans le constructeur de Role ?

Reply:
Noté pour être fait dans un second temps, après avoir défini en termes de langage ubiquitaire ce que sont une organisation et un territoire/structure.